mne/epochs.py

```diff
@@ -2133,14 +2135,15 @@ def __init__(self, data, info, events, tmin=0, event_id=None,
         if data.shape[0] != len(events):
             raise ValueError('The number of epochs and the number of events'
                              'must match')
+        info = deepcopy(info)  # do not modify original info
```

Comment:
Confirmed! Had seen this but forgot after quick local fix ...
On Thu, 21 Apr 2016 at 21:13, Eric Larson <notifications@github.com> wrote:

> In mne/epochs.py
> <https://github.com/mne-tools/mne-python/pull/3155#discussion_r60639114>:
>
> > @@ -2133,14 +2135,15 @@ def __init__(self, data, info, events, tmin=0, event_id=None,
> >          if data.shape[0] != len(events):
> >              raise ValueError('The number of epochs and the number of events'
> >                               'must match')
> > +        info = deepcopy(info)  # do not modify original info
>
> this was a sneaky little bug, too
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/3155/files/dfc1579f08734aa35b053a9d8d811d2ece6b7066#r60639114>
>